contracts/StableJoeStaking.sol

```diff
@@ -224,6 +235,17 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
         emit DepositFeeChanged(_depositFeePercent, oldFee);
     }
 
+    /**
+     * @notice Initialize the Smol Joes address
+     * @param _smolJoes The Smol Joes contract address
+     */
+    function initializeSmolJoes(IERC721Upgradeable _smolJoes) external onlyOwner {
```

Comment:
Is there a reason you only allow setting this value once? From a purely pragmatic point of view if there is some mistake in the `smolJoes` address the contract will be permanently broken and you won't be able to call it again with the correct address.

Reply:
That's not wrong, but as it's a proxy nothing is permanent in any case.